package.json

```diff
@@ -237,7 +237,6 @@
     "text-loader": "^0.0.1",
     "url-search-params": "^0.10.0",
     "us-forms-system": "https://github.com/usds/us-forms-system.git#84887dc7cb0ccee2ae2c8156b5664697c2850b1d",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#36360e480f390f1749f3392a93b143a865437a30",
```

Comment:
Did you mean to remove the schema package?

Reply:
Added this back in. I don't know how it got removed. Definitely a mistake. Thanks for catching this.